api-docs/v2.0/swagger.yml

```diff
@@ -9745,6 +9767,167 @@ components:
           format: float
         legendOrientationThreshold:
           type: integer
+    GeoViewLayer:
```

Comment:
The geo features haven't landed in the `2.0` release branch yet (and aren't planned for 2.0.4), would leave out these additions for now